janusgraph-dist/src/assembly/static/bin/janusgraph.sh

```diff
@@ -213,7 +226,11 @@ clean() {
 
     if cd "$BIN"/../log; then
         rm -f cassandra.log
-        rm -f rexsjanusgraph.log
+        rm -f elasticsearch_deprecation.log
+        rm -f elasticsearch_index_indexing_slowlog.log
+        rm -f elasticsearch_index_search_slowlog.log
+        rm -f elasticsearch.log
```

Comment:
Consider replacing these with `rm -f elasticsearch*.log`. By default Elasticsearch is configured for rolling logs (see [here](https://github.com/elastic/elasticsearch/blob/v5.6.1/distribution/src/main/resources/config/log4j2.properties#L19-L20)).

Reply:
What do you think of `rm -f *.log` or `find -type f -delete`?

Reply:
I'd keep it with the filenames (or prefixes) just for added safety. If you want to go with one I'd choose the first in case it's executed somewhere unexpected.

Reply:
Went with `rm -f elasticsearch*.log`. Updated `rm -f cassandra*.log` since it is also rolling.